buildSrc/build.gradle.kts

```diff
@@ -1,10 +1,9 @@
 
 plugins {
   id("java-library")
-  id("org.jetbrains.kotlin.jvm") version "1.3.11"
+  id("org.jetbrains.kotlin.jvm") version "1.3.30"
```

Comment:
Current plugin is at `1.3.61`. Can we just move up to that?

Reply:
I have no objection, just tried to break as little as possible. Feel free to bump further!